pkg/v1/tkg/test/tkgpackageclient/package_plugin_integration_test.go

```diff
@@ -384,60 +397,178 @@ func cleanup() {
 	Expect(resultImgPullSecret.Error).ToNot(HaveOccurred())
 }
 
+func setUpPrivateRegistry(kubeconfigPath, clusterName string) {
+	By("set up private registry in cluster")
+	kubeCtx := clusterName + "-admin@" + clusterName
```

Comment:
Do we need to add clusterName as config value in test-infra tests in that case? Cause right now it is not provided there, cause cluster name was not mandatory till now.
https://gitlab.eng.vmware.com/TKG/test-infra/-/blob/master/tests/e2e/byot/extensions/test_14_tkg_package_plugin.py#L53

Reply:
Actually, I think I can remove the cluster name here. As I assume the default context would point to the correct cluster for those kubeconfig paths?

Reply:
I think so, but not sure if you need to hardcode kubecontext for Corgi management and workload clusters. Can check once the PR was merged and the tests ran or running tests locally as per [these instructions](https://gitlab.eng.vmware.com/TKG/test-infra/-/tree/master/tests) and see if they work as expected.

How about local integration tests. I mean does running ```ginkgo -tags embedproviders pkg/v1/tkg/test/tkgpackageclient/``` for a hardcoded cluster kubeconfig (in order to not create a cluster from scratch) work now?

Reply:
After second thought, we do need cluster name provided for local testing.

Reply:
Discussed offline with @maralavi , we need clusterName for local testing